backend/api/models/friend.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+import uuid
+
+# Related Model
+from api.models.author import Author
+
+
+# NOTE: When a friend record is created there should be two records made, with each user as the primary friend
+class Friend(models.Model):
+    # Unique Id of this friend record
+    friend_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False, unique=True)
+    
+    # The author whose is primary in this relationship
+    my_id = models.ForeignKey(Author, null=False, on_delete=models.CASCADE, related_name="my_author_id")
```

Comment:
try to be consistent with the naming of same values. with the id of the author, let's use the name `authour_id`